src/request-item/request-item.tsx

```diff
@@ -0,0 +1,54 @@
+import { h } from 'preact'
+import cn from 'classnames'
+import classes from './request-item.module.css'
+import defaultAvatar from './defaultAvatar.jpg'
+import { formatDate } from '../utils'
+
+export type RequestItemProps = h.JSX.HTMLAttributes<HTMLDivElement> & {
+	requestTitle: string
+	requestReplies: number
+	requestDate: Date
+	requesterName: string
+	requesterRep: number
+	requestAmt: number
+	myRequest?: boolean
+	avatar?: string
+	detail?: boolean
+}
+export const RequestItem = ({
+	class: className,
+	...props
+}: RequestItemProps) => (
+	<div class={cn(classes.request, className)}>
+		<div class={props.detail ? classes.detail : ''}>
```

Comment:
Can this be included in the parent `<div>`? It would then look like this:
```tsx
<div class={cn(classes.request, className, props.detail && classes.detail)}>
```